core/designsystem/src/main/java/org/mifos/mobilewallet/mifospay/designsystem/component/MifosUserImage.kt

```diff
@@ -0,0 +1,40 @@
+package org.mifos.mobilewallet.mifospay.designsystem.component
+
+import android.graphics.Bitmap
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.layout.ContentScale
+
+/**
+ * @author pratyush
+ * @since 20/12/2023
+ */
+
+@Composable
+fun MifosUserImage(
```

Comment:
This compose screen should be part of the profile package not designsystem, designsystem is only core building block of the app.